app/Main.hs

```diff
@@ -52,7 +52,7 @@ rules env = do
   create ["archive.html"] $ do
     route idRoute
     compile $ do
-      tutorials <- recentFirst =<< loadAll "tutorials/haskell/*/*.md"
+      tutorials <- recentFirst =<< loadAll "tutorials/(haskell|functional-full-stack)/*/*.md"
```

Comment:
Yeah, the unix-like regex doesn't work. In fact, I figured out it's wrong, it should have been something like `{haskell,functional-full-stack}` (parentheses + commas). I have rebuilt it as fold with a list of programming languages.